src/create_string.cpp

```diff
@@ -3,12 +3,18 @@
 #include <string.h>
 #include "create_string.h"
 
-char* create_string(Local<Value> value) {
-  if (value->IsNull() || !value->IsString()) {
+char* create_string(Nan::MaybeLocal<v8::Value> maybevalue) {
+  v8::Local<v8::Value> value;
+  
+  if (maybevalue.ToLocal(&value)) {
+    if (value->IsNull() || !value->IsString()) {
+      return 0;
+    }
+  } else {
     return 0;
   }
 
-  String::Utf8Value string(value);
+  v8::String::Utf8Value string(value);
   char *str = (char *)malloc(string.length() + 1);
```

Comment:
And it would be better if you made it return a pointer to a heap allocated `Nan::Utf8String`, which can be dereferenced for access to the underlying `char *` and allows you to do `delete` when appropriate. This way you avoid the second memory allocation and string copy.